### PR TITLE
Selenium Test

### PR DESCRIPTION
**Pull request by bhasin85**

### CODE REVIEW

File: selenium/src/test/java/au/org/emii/portal/utils/WebElementUtil.java [hunk 1]
```diff
@@ -315,11 +297,23 @@ public WebElement findElement(By by) {
...
-        element.sendKeys(Keys.ENTER);
+    private void click(By by) {
+        int attempts = 1, totalAttempts = 2;
+        while(attempts <= totalAttempts) {
+            try {
+                WebElement element = findElement(by);
+                Assert.assertNotNull(element);
+                WebDriverWait wait = new WebDriverWait(driver, TIMEOUT);
+                wait.until(ExpectedConditions.elementToBeClickable(element));
+                //element.click();
```
Comment (anguss00) [inline selenium/src/test/java/au/org/emii/portal/utils/WebElementUtil.java]: Extra comment here, can we delete this?

**Comment by anguss00** [comment]
Looks good @bhasin85, aside from one small comment

**Comment by bhasin85** [comment]
Thanks @anguss00

**Comment by bhasin85** [comment]
:+1:

**Pull request merged**